kura/org.eclipse.kura.nm/src/main/java/org/eclipse/kura/nm/configuration/NMSettingsConverter.java

```diff
@@ -123,14 +127,14 @@ public static Map<String, Variant<?>> buildIpv4Settings(NetworkProperties props,
         }
 
         if (ip4Status.equals(KuraIpStatus.ENABLEDLAN)) {
-            settings.put("ignore-auto-dns", new Variant<>(true));
+            settings.put(NM_SETTINGS_IPV6_IGNORE_AUTO_DNS, new Variant<>(true));
```

Comment:
These are ipv4 settings. Create a dedicated different constant.

```suggestion
            settings.put(NM_SETTINGS_IPV4_IGNORE_AUTO_DNS, new Variant<>(true));
```